plugins/mcreator-localization/lang/texts.properties

```diff
@@ -9,6 +9,8 @@ dialog.workspace_selector.donate=Donate
 dialog.workspace_selector.import=Import from file...
 dialog.workspace_selector.new_workspace=New workspace...
 dialog.workspace_selector.open_workspace=Open workspace...
+dialog.workspace_selector.news.loading_failed=Connection error
```

Comment:
You should use same key for both cases here

Reply:
:heavy_check_mark: Fixed